README.md

```diff
@@ -30,14 +30,37 @@ Full documentation [is available here](http://docs.docker.com/swarm/).
 You can download and install from source instead of using the Docker
 image. Ensure you have golang, godep and the git client installed.
 
+Note: Starting from Swarm 0.4 golang 1.4.x or later is required for building, so
```

Comment:
@FreedomWangLi I've created a gist with my suggested changes in it. 

---> https://gist.github.com/moxiegirl/62d41e7d904143e20d49

## Installation for Swarm Users

For installing Swarm for use in your environment, use the Docker Swarm
documentation [on docs.docker.com](http://docs.docker.com/swarm/install-w-machine/).

## Installation for Swarm Developers

Developers should download and install from source rather than using the Docker
image. 


### Prerequisites

Beginning with Swarm 0.4 golang 1.4.x or later is required for building Swarm. 
Refer [to the Go installation page](https://golang.org/doc/install#install)
to download and install the golang 1.4.x or later package.

> **Note**: On Ubuntu 14.04, the `apt-get` repositories install golang 1.2.1 version by
> default. So, do not use `apt-get` but install golang 1.4.x manually using the
> instructions provided on the Go site.

You should also [install
Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and
[godep](https://github.com/tools/godep).

### Clone and build Swarm

Then, install the `swarm` binary in the `$GOPATH/bin` directory.  An easy way to
do this is using the `go get` command.

```bash
$ go get github.com/docker/swarm
```
You can also do this manually using the following commands:

```bash
$ mkdir -p $GOPATH/src/github.com/docker/
$ cd $GOPATH/src/github.com/docker/
$ git clone https://github.com/docker/swarm
$ cd swarm
$ $GOPATH/bin/godep go install .
```

### Use the swarm binary

To test your new Swarm build, you can follow any of the procedures [in
the main documentation](http://docs.docker.com/swarm/). Because you have
installed the `swarm` binary, you can call it directly. This means that installed of calling:

```bash
$ docker swarm create
```

You can call:

```bash
$ swarm create
```